.github/workflows/push.yml

```diff
@@ -0,0 +1,57 @@
+name: Single Build & Test (Push)
+
+# workflow is for a branch push only and ignores master.
+# push to master (which is also pull requet merge) has a more elaborate workflow to run
```

Comment:
nit: type (pull **request**)

Reply:
brrr... I have spellcheck on by default... still missed it.... 
fixed.